ci-operator/step-registry/openshift/microshift/includes/openshift-microshift-includes-commands.sh

```diff
@@ -236,37 +236,41 @@ function ci_custom_link_report() {
         [ -d "${test}" ] || continue
 
         total_scenarios=$((total_scenarios + 1))
-
-        # Find junit file
-        local junit_file=""
-        # RF, ginkgo or conformance
-        if [ -f "${test}/log.html" ]; then
-            junit_file="${test}/junit.xml"
-        elif [ -f "${test}/e2e.log" ]; then
-            junit_file="${test}/junit_01.xml"
-        elif [ -f "${test}/ginkgo-results/test-output.log" ]; then
-            junit_file="$( find "${test}/ginkgo-results" -name "junit_e2e_*.xml" -type f | head -1 )"
-        elif [ -f "${test}/gingko-results/test-output.log" ]; then
-            junit_file="$( find "${test}/gingko-results" -name "junit_e2e_*.xml" -type f | head -1 )"
-        fi
+        scenario_status="pass"
 
         # Determine scenario status
-        local scenario_status="pass"
-        if [ -d "${test}/vms/" ]; then
-          if [ ! -f "${junit_file}" ]; then
-            scenario_status="fail"
-          elif [ -f "${junit_file}" ] && grep -q -E '(failures|errors)="[1-9][0-9]*"' "${junit_file}"; then
+        ## Check Tests Set Up phase result
+        for phase_junit_file in "$(ls ${test}/phase_*/junit.xml)"; do
```

Comment:
Do we need `ls` here? 

```suggestion
        for phase_junit_file in "${test}"/phase_*/junit.xml ; do
```